package/network/utils/comgt/files/ncm.json

```diff
@@ -63,5 +63,21 @@
 		},
 		"connect": "AT*ENAP=1,1",
 		"disconnect": "AT*ENAP=0"
+	},
+	"mtk2": {
+                "initialize": [
+			"AT+CFUN=1"
+		],
+                "configure": [
+			"AT+CREG?",
+			"AT+CGDCONT=1,\\\"${pdptype}\\\",\\\"${apn}\\\",0,0",
+                        "AT+CGACT=1,1"
```

Comment:
I see, that makes sense. Other modems may have similar restrictions, I just always disable the PIN query. Adding a separate configure block is a good idea.